packages/react/src/components/Table/TableHead/FilterHeaderRow/FilterHeaderRow.jsx

```diff
@@ -191,170 +193,169 @@ class FilterHeaderRow extends Component {
+          const filterColumnOptions = (options) => {
+            options.sort((a, b) => {
+              return a.text.localeCompare(b.text, { sensitivity: 'base' });
+            });
+            return options;
+          };
+          const memoizeColumnOptions = memoize(filterColumnOptions); // TODO: this memoize isn't really working, should refactor to a higher column level
+          const isLastColumn = visibleColumns.length - 1 === i;
+          // undefined check has the effect of making isFilterable default to true
+          // if unspecified
+          const headerContent =
+            column.isFilterable !== undefined && !column.isFilterable ? (
+              <div />
+            ) : column.options ? (
+              column.isMultiselect ? (
+                <MultiSelect.Filterable
+                  key={columnStateValue}
+                  className={classnames(
+                    `${iotPrefix}--filterheader-multiselect`,
+                    `${iotPrefix}--filterheader-multiselect__menu--fit-content`,
+                    {
+                      [`${iotPrefix}--filterheader-multiselect__menu--left`]: isLastColumn,
                     }
-                    onChange={(evt) => {
-                      this.setState(
-                        (state) => ({
-                          filterValues: {
-                            ...state.filterValues,
-                            [column.id]: evt.selectedItems.map((item) => item.text),
-                          },
-                        }),
-                        this.handleApplyFilter
-                      );
-                    }}
-                    light
-                    disabled={isDisabled}
-                  />
-                ) : (
-                  <ComboBox
-                    key={columnStateValue}
-                    className={`${iotPrefix}--filterheader-combo`}
-                    id={`column-${i}`}
-                    aria-label={filterText}
-                    translateWithId={this.handleTranslation}
-                    items={memoizeColumnOptions(column.options)}
-                    itemToString={(item) => (item ? item.text : '')}
-                    initialSelectedItem={{
-                      id: columnStateValue,
-                      text: (
-                        column.options.find((option) => option.id === columnStateValue) || {
-                          text: '',
-                        }
-                      ).text, // eslint-disable-line react/destructuring-assignment
-                    }}
-                    placeholder={column.placeholderText || 'Choose an option'}
-                    onChange={(evt) => {
-                      this.setState(
-                        (state) => ({
-                          filterValues: {
-                            ...state.filterValues,
-                            [column.id]: evt.selectedItem === null ? '' : evt.selectedItem.id,
```

Comment:
Was this a bug previously or the result of switching from carbon's combobox to ours?

Reply:
It was a result of switching from carbon's combobox to ours. Not really a drop-in replacement it seemed :-)